recipe/patches/0001-match-naming-convention-used-by-nmake-also-for-windo.patch

```diff
@@ -0,0 +1,45 @@
+From 0e293ca6b9d79303e10a3e6ca526f1cda91add16 Mon Sep 17 00:00:00 2001
+From: "H. Vetinari" <h.vetinari@gmx.com>
+Date: Thu, 21 Jul 2022 23:57:06 +0200
+Subject: [PATCH] match naming convention used by nmake also for windows build
+ from CMake
+
+---
+ CMakeLists.txt | 14 ++++++++++++++
+ 1 file changed, 14 insertions(+)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index d1b120bd..8d80783d 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -358,6 +358,14 @@ ${CMAKE_CURRENT_SOURCE_DIR}/src/webp/types.h")
+ 
+ configure_pkg_config("src/demux/libwebpdemux.pc")
+ 
++if(MSVC)
```

Comment:
```cmake
if(WIN32)
    # Prefix all shared libraries with 'lib'.
    set(CMAKE_SHARED_LIBRARY_PREFIX "lib")
    # Prefix all static libraries with 'lib'.
    set(CMAKE_STATIC_LIBRARY_PREFIX "lib")
endif()
```
Were you looking for something like this? You want all of the libraries to be prefixed with "lib", right?

source: https://stackoverflow.com/a/63471388

Reply:
Yeah, that's nicer, I agree. Googling the right CMake variable is hard.

Reply:
> Googling the right CMake variable is hard.

Agreed.

Reply:
So, are we good to go here now? 🙃